src/matrix/room/timeline/Timeline.js

```diff
@@ -261,6 +261,8 @@ export class Timeline {
             }
             if (contextEvent) {
                 entry.setContextEntry(contextEvent);
+                // emit this change
+                this._remoteEntries.update(entry);
```

Comment:
I'm not sure if this is safe to do... is there a race between sortedArray operations done by `replaceEntries`/`addEntries` and this update :thinking: 
Maybe we need to do:
```js
this._remoteEntries.getAndUpdate(entry, (entry, existingEntry) => existingEntry);
```

Reply:
I'm using this for now:
```js
this._findAndUpdateRelatedEntry(null, entry.id, () => true); 
```